docs/distributions/endeavouros/installation.md

```diff
@@ -13,23 +13,32 @@ You will need:
 2. Boot into the ISO and start the Calamares installer
 
     1. If you're not connected to the internet, connect to it now. Use included GUI config tool to connect to Wi-Fi.
+
+    !!! Note "Issues with Wi-Fi"
+        Some Wi-Fi cards may have issues. If your Wi-Fi tool (e.g., nmcli or wifi-menu) reports that there are no networks found, try reprobing the network card by running `sudo modprobe -r $(lsmod | grep 'brcmfmac' | awk '{print $1}') && sudo modprobe brcmfmac` in a Konsole window.
```

Comment:
```
sudo modprobe -r $(lsmod | grep 'brcmfmac' | awk '{print $1}') && sudo modprobe brcmfmac
```

Why such a complicated command? What's wrong with `sudo modprobe -r brcmfmac_wcc && sudo modprobe -r brcmfmac && sudo modprobe brcmfmac`

Reply:
Pretty sure there are other modules loaded in other than those two so it just throws error when you modprobe -r

Could be wrong though, will need to double check later, getting late for me